From 2057f93d9009576bac66b32999f373eb4c501133 Mon Sep 17 00:00:00 2001
From: haiqiang yao <haiqiangx.yao@intel.com>
Date: Thu, 15 Jan 2015 14:53:52 +0800
Subject: [PATCH 237/255] atomisp.h: add ATOMISP_IOC_S_ARRAY_RESOLUTION

add a new IOCTL to set the sensor's full array size which is used
for crop region setting

Change-Id: I3da6a62f8d674be6e437c9527b9aeee10553453b
Tracked-On:
https://hsdes.intel.com/home/default.html#article?id=1503903826
Signed-off-by: haiqiang yao <haiqiangx.yao@intel.com>
---
 include/linux/atomisp.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/include/linux/atomisp.h b/include/linux/atomisp.h
index 177ec25..d20de5a 100644
--- a/include/linux/atomisp.h
+++ b/include/linux/atomisp.h
@@ -1202,8 +1202,8 @@ struct atomisp_sensor_ae_bracketing_lut {
 #define ATOMISP_IOC_G_INVALID_FRAME_NUM \
 	_IOR('v', BASE_VIDIOC_PRIVATE + 44, unsigned int)
 
-#define ATOMISP_IOC_G_EFFECTIVE_RESOLUTION \
-	_IOR('v', BASE_VIDIOC_PRIVATE + 45, struct atomisp_resolution)
+#define ATOMISP_IOC_S_ARRAY_RESOLUTION \
+	_IOW('v', BASE_VIDIOC_PRIVATE + 45, struct atomisp_resolution)
 
 /* for depth mode sensor frame sync compensation */
 #define ATOMISP_IOC_G_DEPTH_SYNC_COMP \
-- 
1.9.1

